frontend/components/Timeline.js

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from "react";
+import {Interval, TimelineContext} from "../contexts/TimelineContext";
+import DocumentShowcase from "./DocumentShowcase";
+import {TimelineSlider} from "./TimelineSlider";
+import HEAR_CALL_GARDEN from "../images/hear_call_of_the_garden.png";
+import WASHINGTON_BEE_12_3_1910 from "../images/washington_bee_dec_3_1910.png";
+import WASHINGTON_BEE_11_15_1913 from "../images/washington_bee_nov_15_1913.png";
+import {TimelineDropdown} from "./global/TimelineDropdown";
+
+
+export function Timeline() {
+    // will change to make it 2022
```

Comment:
I think our max year should be whatever latest document we have. No need to display a whole bunch of empty intervals! `Todo` for the near future.

Reply:
Thanks for feedback! The max year of the documents we have is 1921. But, our timeline interval show years of 5 (e.g. 1915- 1920, 1920 - 1925), so we set the max year to be 1925. We will make the max year dynamic once we have the backend changes.